react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+
+export default function AppDrawer() {
+  const [open, setOpen] = useState(false);
+
+  function handleClick() {
+    setOpen(!open);
+  }
+
+  const clicked = !open ? 'hidden' : 'modal';
+  const menu = !open ? '' : 'menu-open';
+  const menuLi = !open ? 'hidden' : 'menu-li';
+
+  return (
+    <>
+      <div className=''>
+        <i className="fa-solid fa-bars" onClick={handleClick}></i>
+      </div>
+      <div className='plank'><img src="https://pbs.twimg.com/media/E5zK7rXXMAUdku_.jpg" alt="doge" /></div>
+      <div className={clicked} onClick={() => {
+        setOpen(false);
+      }}>
+      </div>
+      <div className={`menu ${menu}`} onClick={() => {
```

Comment:
This closes the menu if you click anywhere in the menu. The desired behavior is that it only close if you click a specific menu item. Ideally, you would be able to change the contents of the backdrop to whatever the user clicked.

It is a good exercise to see if you can console log the name of the menu item that was clicked.